tests/html/about.html

```diff
@@ -0,0 +1,186 @@
+<html>
```

Comment:
Are you sure `<!DOCTYPE html>` should not be used? Because multi-class will not work in HTML 1.0